main.py

```diff
@@ -22,13 +25,18 @@
         sys.exit(1)
 
     transactions = TransactionCollection()
+    payments = DividendCollection()
 
     if args.transactions_trading212:
         Trading212TransactionsImporter.import_transactions(transactions, args.transactions_trading212)
+        #Trading212DividendsImporter.import_dividends(transactions, args.dividends_trading212)
 
     if args.transactions_binck:
         BinckTransactionsImporter.import_transactions(transactions, args.transactions_binck)
 
+    if args.dividends_trading212:
+       Trading212DividendsImporter.import_dividends(payments, args.dividends_trading212)
+
```

Comment:
If we apply the previous suggestion then this is no longer needed.

Reply:
Now this is not working any more. We need to call the `import_dividends()` function, but we should use the `args.transactions_trading212` argument instead of `args.dividends_trading212`.